### PR TITLE
enable rosa-sts-pl-critical-high-f3 terraform provider job

### PR DESCRIPTION
**Pull request by amalykhi**
https://issues.redhat.com/browse/OCM-4429
Setup New Periodic Prow CI job to build clusters based on Rosa-sts-pl profile

### CODE REVIEW

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 1]
```diff
@@ -135,6 +135,19 @@ tests:
...
     workflow: rhcs-aws-sts
+- as: rosa-sts-pl-critical-high-f3
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
+  steps:
+    cluster_profile: aws-terraform-qe
+    env:
+      CASE_LABEL_FILTER: (Critical,High)&&(day1-post,day2)&&!Exclude
+      CHANNEL_GROUP: candidate
+      CLUSTER_PROFILE: rosa-sts-pl
+      REGION: us-east-1
```
[the remark this answers is not in the document]
Comment (shellymiron) [reply ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:146]: you can use this one: `ap-northeast-1`

File: ci-operator/jobs/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main-periodics.yaml [hunk 2]
```diff
@@ -178,3 +178,93 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build01
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
```
Comment (shellymiron) [inline ci-operator/jobs/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main-periodics.yaml:183]: maybe let's change the hour and dates? not critical though

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 3]
```diff
@@ -135,6 +135,19 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-sts-pl-critical-high-f3
```
Comment (shellymiron) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:138]: maybe let's change the hour and dates? not critical though

File: ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml [hunk 4]
```diff
@@ -135,6 +135,19 @@ tests:
...
+  cron: 38 11 3,6,9,12,15,18,21,23,27,30 * *
```
Comment (shellymiron) [inline ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml:139]: maybe let's change the hour and dates? not critical though

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
@AsherShoshan @shellymiron Fixed your reviews

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3
--


<br class="Apple-interchange-newline">

**Comment by amalykhi** [comment]
/pj-rehearse periodic-ci-terraform-redhat-terraform-provider-rhcs-main-rosa-sts-pl-critical-high-f3

**Comment by amalykhi** [comment]
@vkareh Please review the PR

**Comment by amalykhi** [comment]
@yasun1 please review

**Comment by AsherShoshan** [comment]
@amalykhi leaving the us-east-1 region?

**Comment by amalykhi** [comment]
> @amalykhi leaving the us-east-1 region?

yes, as checked with XueLi PrivateLink is not supported in ap-northeast region)

**Comment by yasun1** [comment]
@amalykhi We are not the approver in this group. Need the persons in [ci-operator/config/terraform-redhat/terraform-provider-rhcs/OWNERS](https://github.com/openshift/release/blob/master/ci-operator/config/terraform-redhat/terraform-provider-rhcs/OWNERS) to give 'OK'. cc @vkareh

**Pull request merged**